### DISCUSSION TITLE
Why lancedb db.py only support create/overwrite for create table?

### DISCUSSION THREAD

**Discussion by brucewkz** [lancedb/lancedb#1499] [Q&A]
db.py   
       See
        ---
        DBConnection.create_table
        """
        if mode.lower() not in ["create", "overwrite"]:
            raise ValueError("mode must be either 'create' or 'overwrite'")

**Comment by wjones127** [comment]
Those are generally the only two options when creating a table:

* `mode="create"` is like `CREATE TABLE ...` in a SQL database. It will fail if the table already exists.
* `mode="overwrite"` is like `CREATE OR REPLACE TABLE ...` in a SQL database. It will succeed even if a table already exists.

If you want to add data to a table, you can use the `add()` method on the table. If you want to modify records, you can use methods like `update()` or `merge_insert()`. You can read more about writing to tables here: https://lancedb.github.io/lancedb/guides/tables/#creating-a-lancedb-table

**Reply by brucewkz** [reply]
> Those are generally the only two options when creating a table:
> 
> * `mode="create"` is like `CREATE TABLE ...` in a SQL database. It will fail if the table already exists.
> * `mode="overwrite"` is like `CREATE OR REPLACE TABLE ...` in a SQL database. It will succeed even if a table already exists.
> 
> If you want to add data to a table, you can use the `add()` method on the table. If you want to modify records, you can use methods like `update()` or `merge_insert()`. You can read more about writing to tables here: https://lancedb.github.io/lancedb/guides/tables/#creating-a-lancedb-table

Got it, pretty thx.
